tyk-pro/configs/tib.conf

```diff
@@ -0,0 +1,35 @@
+{
+ "Secret": "test-secret",
+ "HttpServerOptions": {
+   "UseSSL": true,
+   "CertFile": "./certs/server.pem",
+   "KeyFile": "./certs/server.key"
```

Comment:
Does this mean, we have to ensure these files need to be added?
Can we update the README for this?